File: me/seeking/managers/FileManager.java
package me.seeking.managers;

import net.minecraft.client.Minecraft;

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.List;

/**
 * This file is a part of Seeking Client.
 * @author MiLiBlue
 */
public class FileManager {
    public File path = new File(Minecraft.getMinecraft().mcDataDir.getPath());

    /**
     * 这个方法用来保存文件的
     * @param file
     * @param content
     * @return result, true=succeed, false=failed.
     */
    public boolean saveFile(String file, String content){
        try (FileWriter fileWriter = new FileWriter(path.getPath()+"/"+file)) {
            fileWriter.append(content);
            return true;
        } catch (IOException e) {
            return false;
        }
    }

    /**
     * 这个方法用来加载文件的
     * @param file
     * @return File Content
     */
    public List<String> loadFile(String file){
        try {
            return Files.readAllLines(Paths.get(path.toPath() +"/"+file));
        } catch (IOException e) {
            return null;
        }
    }
}

File: src/main/java/me/seeking/managers/FileManager.kt
package me.seeking.managers

import net.minecraft.client.Minecraft
import java.io.File
import java.io.FileWriter
import java.io.IOException
import java.nio.file.Files
import java.nio.file.Paths

/**
 * This file is a part of Seeking Client.
 * @author MiLiBlue
 */
class FileManager {
    var path = File(Minecraft.getMinecraft().mcDataDir.path)

    /**
     * 这个方法用来保存文件的
     * @param file
     * @param content
     * @return result, true=succeed, false=failed.
     */
    fun saveFile(file: String, content: String?): Boolean {
        try {
            FileWriter(path.path + "/" + file).use { fileWriter ->
                fileWriter.append(content)
                return true
            }
        } catch (e: IOException) {
            return false
        }
    }

    /**
     * 这个方法用来加载文件的
     * @param file
     * @return File Content
     */
    fun loadFile(file: String): List<String>? {
        return try {
            Files.readAllLines(Paths.get(path.toPath().toString() + "/" + file))
        } catch (e: IOException) {
            null
        }
    }
}
